3.workload.md

```diff
@@ -0,0 +1,378 @@
+# 3. Workload
+
```

Comment:
Maybe we can add a `**Target User: Infrastructure Operator**` section here (and to the beginning of every other doc)?